### ISSUE TITLE
Wrong name for the melee weapon of naga ringcaster and zephyr

### ISSUE THREAD

**Issue by rodolphecombe** [wesnoth/wesnoth#9606]
### Language

English at least

### Source

Unit files

### Original Text

        name=chakram_melee
        description=_"chakram"

**Comment by CelticMinstrel** [comment]
The weapon doesn't have any direct real-life counterpart that I'm aware of. The closest I can find is [deer horn knives](https://en.wikipedia.org/wiki/Deer_horn_knives), which is a bladed fist-load weapon with a similar profile. I don't think that name is even remotely suitable for the naga however.

**Comment by rodolphecombe** [comment]
What about the [teeko](https://en.wikipedia.org/wiki/Tekk%C5%8D)?
Sure, naga weapons are not exactly tekko, but they are closer to tekko than to chakram IMHO.

**Comment by babaissarkar** [comment]
FWIW _Chakra_ means "Disc" in Sanskrit (and in Bengali too).

**Comment by CelticMinstrel** [comment]
> Sure, naga weapons are not exactly tekko, but they are closer to tekko than to chakram IMHO.

Except for the fact that tekko is a blunt weapon. The name is also not good.

**Comment by rodolphecombe** [comment]
> Is there an issue with the ranged weapon being a "chakram", or is the issue just that the name is reused for the melee weapon?

The latter. 

> 
> Edit: you can sort-of link to Discord discussions by linking to [the IRC logs](https://irclogs.wesnoth.org/2024/11/%23wesnoth.2024-11-25.log), at least for a given day.

Ah yes, I forgot that.
Here is the [link](https://irclogs.wesnoth.org/2024/11/%23wesnoth.2024-11-25.log). Relevant conversation starts shortly after the beginning.

**Comment by rodolphecombe** [comment]
What about using "war blade"as for the drakes. The weapons look similar to me, at least for the lvl 2

**Comment by CelticMinstrel** [comment]
> This looks a lot like what the Zephyr uses. Why is the name not suitable? Could the Chinese name be used instead?

I don't remember what I was thinking when I wrote the previous post, but the wind and fire wheels are a pair of matched weapons (the wind wheel and the fire wheel), which doesn't seem like it matches with the naga weapons. (I think they do have two? But they're identical.)

Using the Chinese name probably isn't the best either, but… perhaps it's not _as_ bad as writing it in English. There are at least two possible options there as well – in Mandarin it is "fēnghuǒlún", while in Cantonese it is "fung fo leon". Wiktionary didn't include its pronunciation in other Chinese languages, but if someone here speaks Wu or Min or Gan or Xiang Chinese, then using that pronunciation may also be an option.

And then there's also the option to take the Chinese name and mangle it into something that sounds cool.

**Comment by gfgtdf** [comment]
i don't really see what the problem is, it seems like the unit has one weapon that it can both throw and punch the opponents with, so it seems expected that their both attacks have the same name.

**Comment by rodolphecombe** [comment]
No, if you look at the sprite and portrait, it has one throwing weapon, the chakram, that is a full circle, and a melee weapon, whose name is currently also chakram, but which is not a chakram as it is a half circle with a handle and spikes.

**Comment by gfgtdf** [comment]
hmm the unit description says it's a "rejected chakram" tho

> The chakram, a greater blade ring than the chakri, is the signature weapon of the southern Naga and their renowned Ringcasters. Well balanced chakrams are useful for throwing, but the rejected blades make a good fist-load weapon, similar to a sharp-bladed tekko or knuckleduster.

So im still not sure about this ine, but i guess changing the descrption is easier than changing the image at least.
